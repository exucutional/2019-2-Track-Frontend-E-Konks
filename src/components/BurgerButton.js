import React from 'react';
import styled from '@emotion/styled';

const Button = styled.img`
	width: 3em;
	margin-left: 10px;
	&:hover {
		filter: invert(0.5);
	};
	cursor: pointer;
`;

function BurgerButton() {
	return (
		<Button src="http://vcmediapartners.com/media/images/hamburger-white.svg" />
	);
}

export default BurgerButton;
