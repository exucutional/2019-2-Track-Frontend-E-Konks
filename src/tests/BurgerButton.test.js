import React from 'react';
import renderer from 'react-test-renderer';
import BurgerButton from '../components/BurgerButton'

it('renders correctly', () => {
	const tree = renderer
		.create(<BurgerButton/>)
		.toJSON();
	expect(tree).toMatchSnapshot();
});
