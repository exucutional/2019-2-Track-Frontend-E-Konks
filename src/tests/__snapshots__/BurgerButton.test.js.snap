// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<img
  className="css-m3iz7d"
  src="http://vcmediapartners.com/media/images/hamburger-white.svg"
/>
`;
